doc/library/tensor/nnet/conv.txt

```diff
@@ -24,8 +24,14 @@
 
 .. note::
 
-    As of October 21st, 2014, the default GPU image convolution
-    changed: By default, if :ref:`cuDNN <libdoc_cuda_dnn>`
+    As of December 2015, a new conv2d interface has been introduced.
+    :func:`nnet.conv2d <theano.tensor.nnet.conv2d>` defines an
+    abstract theano graph convolution operation
+    (:func:`nnet.abstract_conv.AbstractConv2d <theano.tensor.nnet.abstract_conv.AbstractConv2d>`)
+    that will be replaced by an actual convolution implementation during
+    the optimization phase.
+
+    By default, if :ref:`cuDNN <libdoc_cuda_dnn>`
```

Comment:
Below, I would say "Either CuDNN or the gemm version can be disabled", since the abstract Op does not get optimized to the legacy GPU convolution.
Maybe you can mention (later?) that this is still possible to get the legacy gpuconv2d using the old nnet.conv2d.conv2d() call.